tools/packaging/scripts/lib.sh

```diff
@@ -11,6 +11,8 @@ export tests_repo_dir="$GOPATH/src/$tests_repo"
 
 this_script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
+short_commit_length=10
```

Comment:
Could you put this on a separate commit and explain what this is being used by (`tools/packaging/release/publish-kata-image.sh`?)

Reply:
You'll have to explain to me first :D

Reply:
Updated commit message -- please TAL and let me know if this is okay for you (ACK that I didn't make it a unique commit).